miner/envs/runner/_publish-image.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -eux
+
+if [ -z "$(docker info 2>/dev/null | grep 'Username' | awk '{print $2}')" ]; then
```

Comment:
```suggestion
if [ -z "$(docker info 2>/dev/null | awk '/Username/ {print $2}')" ]; then
```
pretty sure you could also do `docker info --format  SOMETHING` but I guess it is not worth ($$$) figuring out what SOMETHING would be :P